l1-contracts/contracts/state-transition/chain-interfaces/IExecutor.sol

```diff
@@ -32,10 +36,11 @@ struct LogProcessingOutput {
     bytes32 stateDiffHash;
     bytes32 l2LogsTreeRoot;
     uint256 packedBatchAndL2BlockTimestamp;
-    bytes32 blob1Hash;
-    bytes32 blob2Hash;
 }
 
+/// @dev The maximum number of bytes that calldata should be.
+uint256 constant MAX_CALLDATA_SIZE = 128_000;
```

Comment:
The job of this constant is to ensure that in case we choose the "calldata" DA, it indeed does fit into exactly one blob. Probably a more suitable value is `4096 * 31 = 126976`

Reply:
Setting it to anything lower actually puts the server into risk that it creates something of size 121k